core/services/DisplayScreen.py

```diff
@@ -516,57 +516,77 @@ def _event_detection_slider_value_changed(self):
             self.detector.enable_event_detection()
         else:
             self.detector.disable_event_detection()
+            self._update_detected_events('Events')
 
     def _abnormal_activity_slider_value_changed(self):
         value = self.abnormal_activity_slider.value()
         if value == 1:
             self.detector.enable_unusual_activity_detection()
         else:
             self.detector.disable_unusual_activity_detection()
+            self._update_detected_activity('Activity')
```

Comment:
Don't use update_detected_activity() set the text explicitly. Do same for other classifiers

Reply:
@vrajesh978, resolve this. This way you won't have to check whether classifier is ON or not in `_update*` functions. Use `self.qt.set_obj_text()` here itself.

Reply:
Okay